OBAKit/Services/OBARegionValidator.h

```diff
@@ -0,0 +1,28 @@
+//
+//  OBARegionValidator.h
+//  org.onebusaway.iphone
+//
+//  Created by Aaron Brethorst on 8/25/16.
+//  Copyright © 2016 OneBusAway. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <PromiseKit/PromiseKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OBARegionValidator : NSObject
```

Comment:
I bet I can come up with a better class name than this. It was originally semi-relevant but the purpose of the class changed.